src/app/container/refresh-wizard/refresh-wizard.component.html

```diff
@@ -0,0 +1,72 @@
+<div class="modal-body">
+  <p>
+    Select a Quay.io namespace and then click the sync icon next to a repository to automatically register or sync the tool on Dockstore.
+    Requires a Quay.io account linked on Dockstore and also Git build triggers to be present on the repository.
+  </p>
+  <app-loading [loading]="loading$ | async">
+    <mat-card class="alert alert-danger" role="alert" *ngIf="error$ | async as error">
+      <button type="button" class="close" data-dismiss="alert" (click)="clearError()">
+        &times;
+      </button>
+      <p>
+        <mat-icon>warning</mat-icon>
+        The webservice encountered an error.
+      </p>
+      <p class="error-output">
+        {{
+          '[HTTP ' +
+            error.status +
+            '] ' +
+            error.statusText +
+            ': ' +
+            (error.error && error.error.message ? error.error.message : error.error)
+        }}
+      </p>
+    </mat-card>
+    <div *ngIf="organizations$ | async as organizations">
+      <div *ngIf="organizations === undefined">
+        I don't know how you got passed the loading bar
```

Comment:
`past` not `passed`, but I would just get rid of this ngIf and change the one on line 30 to be `*ngIf="!organizations"`

Reply:
Nope, It's purposely split into two for a reason.  Null is an actually problem returned by the call.  Undefined is something wrong with loading bar not showing